### PR TITLE
PWN-3253 - Connect TransactionImageView and TransactionSwapImageView to details bottomSheet

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3253

## Description of Work
Connect TransactionImageView and TransactionSwapImageView to details bottomSheet

## Screenshots
Android Emulator - Pixel_5_API_30:5554 2022-04-19 18-05-40 

Android Emulator - Pixel_5_API_30:5554 2022-04-19 18-05-50 

 Android Emulator - Pixel_5_API_30:5554 2022-04-19 18-07-45

### CODE REVIEW

**Review by kuznetsov — APPROVED** [review]

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -9,4 +9,10 @@
     <dimen name="history_transaction_list_last_item_padding">56dp</dimen>
     <dimen name="history_transaction_item_height">50dp</dimen>
     <dimen name="history_transaction_image_size">50dp</dimen>
+    <dimen name="history_transaction_image_icon_size">18dp</dimen>
```
Comment (1sakov) [inline app/src/main/res/values/dimens.xml:12]: Are you sure it's not 16 or 20 here?

**Review by 1sakov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetFragment.kt [hunk 2]
```diff
@@ -71,6 +74,29 @@ class TransactionDetailsBottomSheetFragment :
         )
     }
 
+    override fun showTransferView(iconRes: Int) = with(binding) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetFragment.kt:77]: can we put @DrawableRes here?

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/res/values/dimens.xml:12]: yes

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetFragment.kt (hunk 2 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/TransactionDetailsBottomSheetFragment.kt:77]: blet =( sorry

**Pull request merged**